.github/workflows/build_and_test_workflow.yml

```diff
@@ -25,18 +25,18 @@ env:
   OSD_SNAPSHOT_SKIP_VERIFY_CHECKSUM: true
 
 jobs:
-  build-lint-test:
+  build-lint-test-linux:
     runs-on: ubuntu-latest
     container:
       image: docker://opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2
       options: --user 1001
-    name: Build and Verify
+    name: Build and Verify on Linux
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Setup Node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
```

Comment:
Just curious to understand why are we bumping the version here?

Reply:
`v2` actions are all Node v12 actions and have been deprecated; `v3` actions are Node v16 based.

https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/